tests/test_yaml_parser.py

```diff
@@ -52,7 +53,7 @@ def test_yaml_parser():
 
 def parse_miopen_yaml1(miopen_yaml, miopen):
   yaml_files = parse_yaml(miopen_yaml, miopen)
-  assert len(yaml_files) == 3
+  assert len(yaml_files) == 4
```

Comment:
add the 4 dict below and compare expected and actual as its done with the previous 3 dicts. To find what that dict looks like u will have to run go_fish.py with a load_job yaml file. Also please add a load_job yaml file to /MITuna/tuna/miopen/yaml_files call it sample4.yaml, this should obviously be a load_jobs yaml

Reply:
currently you can run go_fish will all 3 yaml files from tuna/miopen/yaml_files like:
```
./go_fish.py miopen --yaml miopen/yaml_files/sample1.yaml 
```